linode_api4/groups/linode.py

```diff
@@ -155,6 +157,15 @@ def instance_create(
                region,
                image=image)
 
+        To output the password from the above example:
+            print(password)
+
+        To output the IPv4 address of the new Linode:
```

Comment:
Since the `ipv4` attribute contains a list of all IPv4 addresses assigned to the Linode, could the wording here be updated to reflect that?